single.php

```diff
@@ -109,13 +109,6 @@
 	'comment_field'        => Timber::compile( 'comment_form/comment_field.twig' ),
 	'submit_button'        => Timber::compile( 'comment_form/submit_button.twig' ),
 	'title_reply'          => __( 'Leave your reply', 'planet4-master-theme' ),
-	'fields'               => apply_filters(
```

Comment:
Using this to override the comment form default fields didn't allow us to customise the "Cookies opt-in" field, which is why I had to use a different tactic